lib/parts/auths/apple.dart

```diff
@@ -5,10 +5,12 @@ import 'package:sign_in_with_apple/sign_in_with_apple.dart';
 class AppleAuth extends AuthProvider {
   String _fetchCallbackRoute;
   String Function() _nonce;
+  Api _api;
 
   AppleAuth(Application application) : super(application) {
     _fetchCallbackRoute = application.settings.auths.apple.fetchCallbackRoute;
     _nonce = application.settings.auths.apple.nonce;
+    _api = application.apiLogin ?? application.api;
```

Comment:
Got it, but instead creating a new property inside general application class, you can use the private _AuthApple class (middleware/settings.dart) for this matter.

Take the line above as an example, private variable _nonce is setted from that class.

Reply:
ready